frontend/src/model/beta/Releases.tsx

```diff
@@ -30,15 +31,20 @@ export default function Releases({ model }: { model: ModelInterface }) {
 
   function handleDraftNewReleaseClose() {
     setOpenDraftNewRelease(false)
+    setLoading(false)
   }
 
+  function handleButtonClick() {
+    setOpenDraftNewRelease(true)
+    setLoading(true)
+  }
   return (
     <Box sx={{ maxWidth: '900px', mx: 'auto', my: 4 }}>
       <Stack spacing={4}>
         <Box sx={{ textAlign: 'right' }}>
-          <Button variant='outlined' onClick={() => setOpenDraftNewRelease(true)} disabled={!model.card}>
+          <LoadingButton variant='outlined' onClick={handleButtonClick} loading={loading} disabled={!model.card}>
```

Comment:
This LoadingButton isn't needed as it's only being used to open a dialog. There isn't anything we're waiting for here.